packages/compiler/test/checker/templates.test.ts

```diff
@@ -124,7 +124,7 @@ describe("compiler: templates", () => {
     const { A } = (await testHost.compile("main.tsp")) as { A: Model };
     const a = A.properties.get("a")!;
     strictEqual(a.type.kind, "Scalar");
-    strictEqual((a.type as Model).name, "string");
+    strictEqual(a.type.name, "string");
```

Comment:
Ooh, this means the places where did `strictEqual(a.kind, "b" as const)` can be more natural `strictEuqal(a.kind, "b")` now too, right?

Reply:
I think so

Reply:
there is also a lot of place with the lib definition for example where we can use the new `<const T>` template param so we don't need to use `as const`